src/app/explore/explore.component.html

```diff
@@ -15,7 +15,7 @@
       [beneficiaryOptions]="beneficiaryOptions"
       [locationOptions]="locationOptions"
       [fundingOptions]="fundingOptions"
-      [selectedSortByOption]="searchService.getSelectedSortLabel()"
+      [selectedSortByOption]="searchService.selectedSortLabel"
```

Comment:
I'm confused by this - it looks like the prop is typed as `'Most raised' | 'Match funds remaining' | 'Relevance' | null` but you're now passing a function rather than a string or null - how does that work?
https://github.com/thebiggive/components/blob/7aa4dfad27472c6aa070f6ed90148e7adfd77f26/src/components/biggive-campaign-card-filter-grid/biggive-campaign-card-filter-grid.tsx#L95